pkg/snapshot/restorer/restorer.go

```diff
@@ -583,18 +599,32 @@ func getEventsFromDeltaSnapshot(store snapstore.SnapStore, snap snapstore.Snapsh
 }
 
 // getEventsDataFromDeltaSnapshot fetches the events data from delta snapshot from snap store.
-func getEventsDataFromDeltaSnapshot(store snapstore.SnapStore, snap snapstore.Snapshot) ([]byte, error) {
-	rc, err := store.Fetch(snap)
+func (r *Restorer) getEventsDataFromDeltaSnapshot(store snapstore.SnapStore, snap snapstore.Snapshot) ([]byte, error) {
```

Comment:
Any particular reason to add this function as a member under `Restorer` object? This has led to subsequent changes in other function signatures as well, like `getEventsFromDeltaSnapshot`, `makeWALAndSnap` and `makeDB`

Reply:
yes , reason is I need to print the log for decompression time . you can see [here](https://github.com/gardener/etcd-backup-restore/pull/293/files#diff-9547d8cc0c155be97c8e3a057b9351b6bf83108543f957fb8bac1adf111ccfaaR165)